kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -1,6 +1,7 @@
 import logging
 
 from airflow import DAG
+from airflow.contrib.kubernetes.secret import Secret
```

Comment:
Is it working with Airflow 2.x? I feel like this import is from 1.x only.
EDIT: `from airflow.kubernetes.secret import Secret` is valid for 2.x

Reply:
Done